pages/getting-started/understand-odata-in-6-steps.html

```diff
@@ -247,7 +247,9 @@ <h4 class="modal-title" id="step-collection-modal-label">Response</h4>
 {% endhighlight %}
             </p></div>
 <div class="modal-footer">
+<button type="button" class="btn btn-secondary btn-copy-code" onclick="copyCode()">Click to Copy Response</button>
```

Comment:
```suggestion
<button type="button" class="btn btn-secondary btn-copy-code" onclick="copyCode()">Copy</button>
```
Same way we don't say "Click to Close", I believe "Copy" should suffice - or "Copy to clipboard" (if we need to be specific)

Reply:
Used `Copy`